boards/esp32-ethernet-kit-v1_0/doc_common.txt

```diff
@@ -90,4 +90,18 @@ To use the JTAG interface, the `esp_jtag` module must be used to disable the `SP
 USEMODULE=esp_jtag make flash BOARD=esp32-ethernet-kit-v1_X ...
 Furthermore the function switches (DIP switches) for the JTAG signals must be set to ON.
 
+To flash using OpenOCD, install the [openocd-esp32](https://github.com/espressif/openocd-esp32) fork.
+Then flash with PROGRAMMER set:
+```
+export OPENOCD="~/openocd-esp32/src/openocd -s ~/openocd-esp32/tcl"
```

Comment:
Can't the `src` part be added to an openocd .cfg file?

Reply:
This is the installation location of the openocd-esp32 code, which can be chosen by the user.
The -s flag is the root location of its config files (can also be env variable https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/jtag-debugging/#building-openocd-from-sources), the board cfg file is inside there.